src/main/java/com/example/demo/services/IBookingService.java

```diff
@@ -0,0 +1,17 @@
+package com.example.demo.services;
+
+import com.example.demo.entities.BookingEntity;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+
+import java.net.URI;
+
+public interface IBookingService {
```

Comment:
Interfaces are quite optional in Spring - it is often easier to ignore them until actually needed as Spring can inject an autowired bean directly without the need for an interface